.github/pull_request_template.md

```
@@ -0,0 +1,61 @@
+- [ ] Code refactoring
+
+## Related Issues
+
+<!-- Link to related issues -->
+Fixes #(issue number)
+
+## Changes Made
+
+<!-- List the specific changes made in this PR -->
+
+- 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)